src/go/k8s/.golangci.yml

```diff
@@ -80,7 +80,6 @@ linters:
     - nestif
     - prealloc
     - testpackage
-    - wsl
```

Comment:
Please don't remove, but move it to line after 84

Reply:
it's actually already there :)

Reply:
Ok, thanks